frontend/packages/shared/src/api/mutations.ts

```diff
@@ -192,6 +192,5 @@ export const deleteCodeListForOrg = async (org: string, codeListId: string): Pro
 export const uploadCodeListForOrg = async (org: string, payload: FormData): Promise<CodeListsResponse> => post(orgCodeListUploadPath(org), payload);
 
 // Organisation text resources:
-// Todo: Replace these with real API calls when endpoints are ready. https://github.com/Altinn/altinn-studio/issues/14503
-export const createTextResourcesForOrg = async (org: string, language: string): Promise<ITextResourcesWithLanguage> => Promise.resolve(textResourcesMock); // Todo: Replace with real API call when endpoint is ready. https://github.com/Altinn/altinn-studio/issues/14503
-export const updateTextResourcesForOrg = async (org: string, language: string, payload: KeyValuePairs<string>): Promise<ITextResourcesWithLanguage> => Promise.resolve(textResourcesMock); // Todo: Replace with real API call when endpoint is ready. https://github.com/Altinn/altinn-studio/issues/14503
+export const createTextResourcesForOrg = async (org: string, language: string): Promise<ITextResourcesWithLanguage> => post<ITextResourcesWithLanguage, null>(orgTextResourcesPath(org, language), null);
```

Comment:
I noticed that this post request has no payload. You have set the data parameters to `null`, so maybe it's coming in a later PR?

It seems like the endpoint expects a payload of this type:

```suggestion
export const createTextResourcesForOrg = async (org: string, language: string, payload: ITextResourcesWithLanguage): Promise<ITextResourcesWithLanguage> => post<ITextResourcesWithLanguage, ITextResourcesWithLanguage>(orgTextResourcesPath(org, language), payload);
```

Reply:
Well spotted! That was clumsy of me. Based on some conversation we had with the team a while ago, I thought the endpoint didn't expect any payload and that it would just create a resource file with an empty list, but I see that's not the case. I guess we need to create the data structure on the frontend instead. I'll fix that and update the pull request. Thanks!

Reply:
I see several problems with the `useCreateTextResourcesForOrgMutation` hook and its test, so I'm going to create a separate pull request to fix it. We'll need to merge in that one first.

Reply:
Here is the pull request: https://github.com/Altinn/altinn-studio/pull/15141

Reply:
Reviewed and merged ✅